opentasks/src/main/java/org/dmfs/tasks/utils/DarkenedColor.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.tasks.utils;
+
+import org.dmfs.android.bolts.color.Color;
+
+
+/**
+ * {@link Color} to get a darkened version of an original color dynamically.
+ * (Typically to then use as status bar color.)
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class DarkenedColor implements Color
```

Comment:
I guess this is primarily supposed to be used as a decorator. Consider calling it just `Darkened`.

Reply:
Right, it is a decorator, I've changed the name.
Also imported latest color-bolts with `ValueColor` and using that in the ctor now.